server/routes/api.js

```diff
@@ -110,4 +109,46 @@ router.patch('/cards', (req, res) => {
     }
 })
 
+/*
+API call to get a single column from the neighborhood list using a constraint array.
+req: 1) the string name of the column to get data from.
+    2) a constraint_array of the same form as the api/getFilteredData endpoint.
+res: an array of numbers from the specified column with the specified constraints.
+*/
+router.post('/column', (req, res) => {
+    console.log("api/column call:")
+    console.log(req.body)
+
+    constraint_array = req.body.constraint_array;
+    column = req.body.column_name;
+
+    //determine the column number based on the column name
+    column_num = -1;
+    switch(column){
+		case "Median_House_Value": column_num = 1; break;
+		case "Median_Income": column_num = 2; break;
+		case "Median_Age": column_num = 3; break;
+		case "Tot_Rooms": column_num = 4; break;
+		case "Tot_Bedrooms": column_num = 5; break;
+		case "Population": column_num = 6; break;
+		case "Households": column_num = 7; break;
+		case "Latitude": column_num = 8; break;
+		case "Longitude": column_num = 9; break;
+		case "Distance_to_coast": column_num = 10; break;
+		case "Distance_to_LA": column_num = 11; break;
+		case "Distance_to_SanDiego": column_num = 12; break;
+		case "Distance_to_SanJose": column_num = 13; break;
+		case "Distance_to_SanFrancisco": column_num = 14; break;
+		case "ID": column_num = 15; break;
```

Comment:
Is there any reason to graph by ID?

Reply:
I added ID access so that this API call can make use of it for potential future development. I don't think we'll ever get ID, but I don't see why we shouldn't allow it. We might be able to use this api call for future analytics